mmv1/products/discoveryengine/DataStore.yaml

```diff
@@ -0,0 +1,168 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'DataStore'
+description: |
+  Vertex AI Search and Conversation data store is a collection of websites and
```

Comment:
Nit: Potentially confusing to see "Vertex AI Search and Conversation" mentioned immediately in the description when the resource is called "discoveryengine". Might want to tweak to at least draw the connection between the two names.

Reply:
Thanks for your concerns. This product was renamed for several times. I updated the description to make it less confusing.